pkg/commands/import/import_test.go

```diff
@@ -786,122 +786,117 @@ Importing ClusterBuilder 'default'...
-kind: ClusterStore
-metadata:
-  annotations:
-    kpack.io/import-timestamp: "2006-01-02T15:04:05Z"
-    kubectl.kubernetes.io/last-applied-configuration: '{"kind":"ClusterStore","apiVersion":"kpack.io/v1alpha2","metadata":{"name":"store-name","creationTimestamp":null},"spec":{"sources":[{"image":"default-registry.io/default-repo@sha256:buildpack-image-digest"}],"serviceAccountRef":{"namespace":"some-namespace","name":"some-serviceaccount"}},"status":{}}'
-  creationTimestamp: null
-  name: store-name
-spec:
-  serviceAccountRef:
-    name: some-serviceaccount
-    namespace: some-namespace
-  sources:
-  - image: default-registry.io/default-repo@sha256:buildpack-image-digest
-status: {}
----
-apiVersion: kpack.io/v1alpha2
-kind: ClusterStack
-metadata:
-  annotations:
-    kpack.io/import-timestamp: "2006-01-02T15:04:05Z"
-  creationTimestamp: null
-  name: stack-name
-spec:
-  buildImage:
-    image: default-registry.io/default-repo@sha256:build-image-digest
-  id: stack-id
-  runImage:
-    image: default-registry.io/default-repo@sha256:build-image-digest
-  serviceAccountRef:
-    name: some-serviceaccount
-    namespace: some-namespace
-status:
-  buildImage: {}
-  runImage: {}
----
-apiVersion: kpack.io/v1alpha2
-kind: ClusterStack
-metadata:
-  annotations:
-    kpack.io/import-timestamp: "2006-01-02T15:04:05Z"
-  creationTimestamp: null
-  name: default
-spec:
-  buildImage:
-    image: default-registry.io/default-repo@sha256:build-image-digest
-  id: stack-id
-  runImage:
-    image: default-registry.io/default-repo@sha256:build-image-digest
-  serviceAccountRef:
-    name: some-serviceaccount
-    namespace: some-namespace
-status:
-  buildImage: {}
-  runImage: {}
----
-apiVersion: kpack.io/v1alpha2
-kind: ClusterBuilder
-metadata:
-  annotations:
-    kpack.io/import-timestamp: "2006-01-02T15:04:05Z"
-    kubectl.kubernetes.io/last-applied-configuration: '{"kind":"ClusterBuilder","apiVersion":"kpack.io/v1alpha2","metadata":{"name":"clusterbuilder-name","creationTimestamp":null},"spec":{"tag":"default-registry.io/default-repo:clusterbuilder-clusterbuilder-name","stack":{"kind":"ClusterStack","name":"stack-name"},"store":{"kind":"ClusterStore","name":"store-name"},"order":[{"group":[{"id":"buildpack-id"}]}],"serviceAccountRef":{"namespace":"some-namespace","name":"some-serviceaccount"}},"status":{"stack":{}}}'
-  creationTimestamp: null
-  name: clusterbuilder-name
-spec:
-  order:
-  - group:
-    - id: buildpack-id
-  serviceAccountRef:
-    name: some-serviceaccount
-    namespace: some-namespace
-  stack:
-    kind: ClusterStack
-    name: stack-name
-  store:
-    kind: ClusterStore
+			const resourceYAML = `- apiVersion: v1
+  data:
+    image: default-registry.io/default-repo@sha256:lifecycle-image-digest
+  kind: ConfigMap
+  metadata:
+    annotations:
+      kpack.io/import-timestamp: "2006-01-02T15:04:05Z"
+    creationTimestamp: null
+    name: lifecycle-image
+    namespace: kpack
+- apiVersion: kpack.io/v1alpha2
```

Comment:
For yaml we should keep the old format of it outputting multiple documents. It's a common practice to pipe the output of one command into another like `kp import descriptor.yaml --output yaml | kubectl apply -f -`, which would fail if it only outputs a single document with an array instead of multiple documents